### PR TITLE
📝  README: Adding Project Removal Section

### PR DESCRIPTION
**Pull request by paOmer**
Adding "How To Remove The Local Project" section to the README file.
Suggested by @kasemAlem via project Slack channel.

You can view the file [here](https://github.com/paOmer/Beyond-07-team-4/blob/readme_update/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,23 @@ Afterwards, the web application will be presented on http://localhost:8000
 
 ---
 
+## How To Remove The Local Project
```
Comment (taljacob2) [inline README.md:89]: I think you can edit this to `## How To Remove The Project`.

File: README.md [hunk 2]
```diff
@@ -86,6 +86,23 @@ Afterwards, the web application will be presented on http://localhost:8000
...
+    ```sh
+    vagrant destroy
+    ```
+    After running the last command, the terminal will prompt the following:
+    ```sh
+    default: Are you sure you want to destroy the 'default' VM? [y/N]
+    ```
+    Type ` y `
+
+- #### Delete The Local Project Root Directory
```
Comment (taljacob2) [inline README.md:102]: I think you can edit this to `#### Delete The Project Root Directory`.

**Review by taljacob2 — APPROVED** [review]
These are good additions.
Please view my suggestions.

File: README.md (hunk 1 above)
Comment (paOmer) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (paOmer) [reply README.md]: Sounds good, changed it

**Review by ShirleyFichman — APPROVED** [review]
Great addition!

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**